### ISSUE TITLE
Explain what Project Hamster is about

### ISSUE THREAD

**Issue by jtojnar** [elbenfreund/hamster-website#2]
When 0.10.0 release was [announced on Reddit](https://www.reddit.com/r/linux/comments/4txtti/hamstergtk_0100_released/), people complained that it is not clear what the project is about.

It should be described either in header or on the front page (above posts?).

**Comment by elbenfreund** [comment]
My idea is to create a seperate "static page" that will try to answer this.
Thank you for creating the issue. Its not the most urgent item on my agenda, but something I want to fix within the next weeks.

**Comment by jtojnar** [comment]
Yes, having a separate page to explain the project in detail would be nice but it would be also good to have a simple explanation on every page. Simple tagline would suffice:

![hamster-tagline](https://cloud.githubusercontent.com/assets/705123/19271009/1f5ff7ee-8fc3-11e6-8aa1-7c3445e06804.png)

**Comment by elbenfreund** [comment]
Any suggestions for the tagline? Keep in mind that this would be for the entire project hamster.
Something like "The free and open time tracking toolkit." ?

**Comment by jtojnar** [comment]
That would work for me, though it might be too long to fit in the navbar.
